### PR TITLE
feat(packages/sui-i18n): import node-polyglot 0.43 to our package

### PR DESCRIPTION
**Pull request by jordevo**
## Description
Import `node-polyglot 0.43` code to our package.

Polyglot adapter is already being fully tested:
https://github.com/SUI-Components/sui/blob/master/packages/sui-i18n/test/polyglotSpec.js#L53 

## Related Issue
Implements #1468 .

### CODE REVIEW

File: packages/sui-i18n/src/adapters/polyglot.js [hunk 1]
```diff
@@ -1,4 +1,222 @@
-import Polyglot from 'node-polyglot'
+// BSD 2-Clause License
+
+// Copyright (c) 2012, Airbnb
+// All rights reserved.
```
Comment (jordevo) [inline packages/sui-i18n/src/adapters/polyglot.js:4]: ℹ️  license notice kept here as per:

https://github.com/airbnb/polyglot.js/blob/master/LICENSE

> 1. Redistributions of source code must retain the above copyright notice, this
>    list of conditions and the following disclaimer.

Comment (oegea) [reply packages/sui-i18n/src/adapters/polyglot.js:4]: 👉 **A -personal- suggestion regarding this:** 

Would it make sense to have a specific file to maintain all third-party license notices? Relating each notice to the specific dependency that requires it.

**Review by oegea — APPROVED** [review]

File: packages/sui-i18n/src/adapters/polyglot.js (hunk 1 above)
Comment (jordevo) [reply packages/sui-i18n/src/adapters/polyglot.js:4]: good point! not sure what's the best way to deal with this in our packages.

Comment (rmoralp) [reply packages/sui-i18n/src/adapters/polyglot.js:4]: We could open an issue to discuss this point and avoid blocking this PR. What do you think?

**Review by rmoralp — APPROVED** [review]

File: packages/sui-i18n/src/adapters/polyglot.js (hunk 1 above)
Comment (jordevo) [reply packages/sui-i18n/src/adapters/polyglot.js:4]: totally agree! will do that then 👍

**Pull request merged**